lib/project/pro_motion/table.rb

```diff
@@ -1,7 +1,8 @@
 module ProMotion
   class TableScreen < TableViewController
-    # For some reason, this empty class definition is required
-    # or we get a NoMethodError with the `searchable`
-    # class method call. O_o
+    # Don't call super -- would result in two on_load calls to the cell.
+    def on_cell_created(cell, data)
+      build(cell)
+    end
```

Comment:
In this case, we don't want to call super, because we're handling the on_load ourselves (ostensibly -- for some reason it doesn't seem that `build()` does call on_load`).

We won't be overriding too often, but I want that data available for when we do, since it would be relevant to what the dev is trying to do most likely.